manifests/config.pp

```diff
@@ -21,6 +21,16 @@
     target => $pulp::ca_cert,
   }
 
+  unless $pulp::manage_httpd {
+    file {'/etc/httpd/conf.d/pulp.conf':
```

Comment:
This should probably just go in apache.pp and then you can use and `else` to deploy this instead of sticking it here with an unless.